source/include/VirtualInstrument.h

```diff
@@ -140,11 +145,10 @@ class VIClump : public FunctionClump
     VirtualInstrument*  TopVI()      {
         VIClump *caller = this;
         do {
-            while (caller->_waitingClumps)
```

Comment:
The old logic was wrong because it did not handle the Trigger'd clumps that weren't actively being waited on.